ibc-clients/README.md

```diff
@@ -25,7 +25,6 @@ IBC light clients:
 
 - [ibc-client-tendermint-types](./ics07-tendermint/types): Data Structures
 - [ibc-client-tendermint](./ics07-tendermint): Implementation
-- [ibc-client-tendermint-cw](./../cosmwasm/ibc-clients/ics07-tendermint): CosmWasm Contract
```

Comment:
Let's add something here instead and bring this to attention that the implementation of ICS-07 CosmWasm contract lives under the cosmwasm-ibc repository. And include a link too.

Reply:
[32608ee](https://github.com/cosmos/ibc-rs/pull/1315/commits/32608eef115ba378a160a7c40c0549510fd450ca)